src/js/page/ui/main-menu.js

```diff
@@ -57,6 +57,11 @@ export default class MainMenu {
     this._loadFileInput.click();
   }
 
+  setPasteInput(value) {
+    this._pasteInput.value = value;
+    this._pasteInput.dispatchEvent(new Event('input'));
```

Comment:
Not sure about using `Event`; `nanoevents` are used everywhere else.

Reply:
It's a little bit of a hack, but if an event is going to be used here, it's the right kind of event, since this is an `HTMLElement`.